<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>interface</class>
 <widget class="QWidget" name="interface">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>420</width>
    <height>284</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>420</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>420</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string notr="true">Ultracopier</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/resources/icon.png</normaloff>:/resources/icon.png</iconset>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QWidget" name="widget" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: qlineargradient(spread:pad, x1:0, y1:0, x2:1, y2:0, stop:0 rgba(230, 240, 255, 255), stop:0.5 rgba(166, 199, 230, 255), stop:1 rgba(3, 40, 84, 255));</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QLabel" name="label_main">
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 0, 0, 0);
font: 12pt &quot;Sans Serif&quot;;</string>
        </property>
        <property name="text">
         <string notr="true">Copying 0 item (0KB)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label_2">
        <property name="minimumSize">
         <size>
          <width>51</width>
          <height>40</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>51</width>
          <height>40</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">background-color: rgba(255, 0, 0, 0);</string>
        </property>
        <property name="pixmap">
         <pixmap resource="resources.qrc">:/resources/icon-top.png</pixmap>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>0</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(255, 255, 255);</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>14</number>
      </property>
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>14</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>14</number>
      </property>
      <item>
       <widget class="QLabel" name="text">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>28</height>
         </size>
        </property>
        <property name="text">
         <string notr="true">from &lt;b&gt;%1&lt;/b&gt; (%2) to &lt;b&gt;%3&lt;/b&gt; (%4)&lt;br /&gt;
%5</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QWidget" name="details" native="true">
        <layout class="QFormLayout" name="formLayout">
         <property name="labelAlignment">
          <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
         </property>
         <property name="verticalSpacing">
          <number>2</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>Name:</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_file">
           <property name="text">
            <string notr="true">file</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>From:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="label_from">
           <property name="text">
            <string notr="true">&lt;b&gt;%1&lt;/b&gt; (%2)</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>To:</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QLabel" name="label_to">
           <property name="text">
            <string notr="true">&lt;b&gt;%1&lt;/b&gt; (%2)</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Time remaining:</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QLabel" name="label_remaining_time">
           <property name="text">
            <string notr="true">About X minutes</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Items remaining:</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QLabel" name="label_items">
           <property name="text">
            <string notr="true">1 252 (13GB)</string>
           </property>
          </widget>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Speed:</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QLabel" name="label_speed">
           <property name="text">
            <string notr="true">52MB/second</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QProgressBar" name="progressBar">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>14</height>
         </size>
        </property>
        <property name="maximum">
         <number>65535</number>
        </property>
        <property name="textVisible">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="minimumSize">
      <size>
       <width>420</width>
       <height>42</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>420</width>
       <height>42</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(237, 237, 237);</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="leftMargin">
       <number>20</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>20</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QToolButton" name="more">
        <property name="text">
         <string>More information</string>
        </property>
        <property name="icon">
         <iconset resource="resources.qrc">
          <normaloff>:/resources/arrow-down.png</normaloff>:/resources/arrow-down.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
        <property name="toolButtonStyle">
         <enum>Qt::ToolButtonTextBesideIcon</enum>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>123</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item>
       <widget class="QToolButton" name="toolButton">
        <property name="text">
         <string notr="true">...</string>
        </property>
        <property name="popupMode">
         <enum>QToolButton::InstantPopup</enum>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="Cancel">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
  <action name="actionAddFile">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add file</string>
   </property>
  </action>
  <action name="actionAddFolder">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Add folder</string>
   </property>
  </action>
  <action name="actionAddFileToMove">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Move file(s)</string>
   </property>
  </action>
  <action name="actionAddFolderToMove">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Move folder</string>
   </property>
  </action>
  <action name="actionAddFileToCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Copy file(s)</string>
   </property>
  </action>
  <action name="actionAddFolderToCopy">
   <property name="icon">
    <iconset>
     <normaloff>:/resources/add.png</normaloff>:/resources/add.png</iconset>
   </property>
   <property name="text">
    <string>Copy folder</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>Cancel</sender>
   <signal>clicked()</signal>
   <receiver>interface</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>388</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>523</x>
     <y>498</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
